src/main/java/seedu/address/model/ReadOnlyTrackIter.java

```diff
@@ -14,4 +15,11 @@
      */
     ObservableList<Contact> getPersonList();
 
+    /**
+     * Returns an unmodifiable view of the modules list.
+     * This list will not contain any duplicate modules.
+     *
+     * @return
+     */
```

Comment:
Need to fill in return value here

Reply:
Okay I forgot. Actually the original code doesn't explain the returned result too (maybe too obvious?)

Reply:
Fixed in the latest commit